CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 - Fix potential performance and stability issues from OpenBLAS oversubscription by limiting nested parallelism in [#87](https://github.com/lemma-osu/sklearn-raster/pull/87)
 - Fix slow performance when `nodata_input` contains explicit or inferred missing NoData values
 - When `output_sizes` is required but not provided to `FeatureArray.apply_ufunc_across_features`, a helpful error message is raised
+- Fix error that prevented using a boolean scalar `nodata_input` with a boolean feature array
```

Comment:
Was this really an error?  I thought it was initially by design to not allow booleans.  This is minor, so feel free to ignore.

Reply:
I went back and forth on where to put that, but decided on a fixed error just because the inconsistency between `nodata_input=[False]` and `nodata_input=False` definitely wasn't intended.